### PR TITLE
TensorBoard 1.5.0

### PR DESCRIPTION
**Pull request by nfelt**
The 1.5.0 release series and beyond will use the new PyPI package name of just plain `tensorboard`.

### CODE REVIEW

**Comment by nfelt** [comment]
@jart any suggestions for what we should do about Travis here?  It's failing because it can't run TF 1.5.0rc1 (which I've pegged this branch to in .travis.yml) on the Ubuntu 14.04 build machine.

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
 
 env:
-  - BAZEL=0.5.4 TF=NIGHTLY
```
Comment (jart) [inline .travis.yml]: We can keep this pinned to nightly and update the branch later, once we fix we fix TensorFlow.

**Comment by zihaolucky** [comment]
Thanks @jart, is this version will change the way we build TensorBoard with Bazel? It's not clear to me how db helps in decoupling, could you explain to me? Thank you😊

My current solution on dmlc/tensorboard has switched to this repo, https://github.com/dmlc/tensorboard/blob/master/installer.sh. We still use pure Python to write `event` files, but the `tensorboard` binary is depends on this repo and TensorFlow, as the bazel build file still rely on it.

**Comment by jart** [comment]
Build is stable and hasn't really changed. Only potential break is we didn't have time to write a Pillow fallback in our TF PNG encoder. We are planning to do this, but can do it very soon if you tell us you need it.

> It's not clear to me how db helps in decoupling

Do you remember how you reverse-engineered Google's binary records file format? It was an excellent hack, but ideally hacks shouldn't stand between our users and their data. Also consider how, even after you accomplished that, you still needed to depend on protobuf, which is 8MB and isn't always guaranteed to go fast depending on how it's installed.

Not everyone agrees on TensorFlow, but there might be opportunity for agreement on SQLite. It's distributed as single .c file with a 400kB footprint that offers 80% the value of most databases. If our experiment works, other frameworks like MXNet could embed SQLite to export tensors directly to TensorBoard.

Please note event logs will be supported forever.

**Comment by zihaolucky** [comment]
@jart Thank you.

> If our experiment works, other frameworks like MXNet could embed SQLite to export tensors directly to TensorBoard.

So other frameworks could write to db via SQLite and TensorBoard could directly reach the data? Great idea! Then how about the backend part, which reads the `events` file(and maybe SQLIte file later)?

**Comment by jart** [comment]
What would likely happen is `--logdir` would spawn a thread that loads events into a /tmp DB and everything works the same. Then add a `--db` flag so users can opt-in to grouping runs by experiment, project, and user. In some cases it might be possible for all that to exist in just one `.sqlite` file.

**Comment by zihaolucky** [comment]
Agree. +1

**Comment by nfelt** [comment]
PTAL - removed the PyPI package rename for now, we'll likely defer that to 1.6.

**Comment by nfelt** [comment]
PTAL, added release notes in RELEASE.md

**Pull request merged**